src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -120,6 +125,13 @@ private void setFilingApiData(
     setPaymentData(userSubmission, transaction, passThroughTokenHeader, logMap);
 
     if (submissionDto.isForUpdate()) {
+      var hashHelper = new HashHelper(salt);
+      try {
+        String digest = hashHelper.encode("1234567890");
+        ApiLogger.info("Digest " + digest);
```

Comment:
we should use the ApiLogger.infoContext and errorContext methods where possible and use the requestId for the context. This will help us to follow a user journey through different log files.

Reply:
Will do. It's a placeholder for now. We'll be using the HashHelper to collate data from the previous registration with the update submission.